web/libs/editor/src/core/DataValidator/ConfigValidator.js

```diff
@@ -222,6 +222,21 @@ const validateParentTag = (element, model) => {
   return errorBuilder.parentTagUnexpected(model.name, 'parent', element.tagName, model.properties.parentTypes);
 };
 
+/**
+ * Validates if visual tags have name attribute
+ * @param {Object} element
+ */
+const validateVisualTags = (element) => {
+  const visualTags = ['Collapse', 'Filter', 'Header', 'Style', 'View'];
```

Comment:
As potential improvement we may try to form this list from the information from `Registry`. But in that case we need new mixin like `BaseVisualTag` with volatile `isVisualTag: true` and mix it into all Visual tags. 
But it may be that I'm overthinking.

Reply:
Oh. We also may just use `model` as it done in other validating functions. And check its `isVisualTag` when we add it to them.